server/config/config_test.go

```diff
@@ -39,6 +36,14 @@ var _ = Suite(&testConfigSuite{})
 
 type testConfigSuite struct{}
 
+func (s *testConfigSuite) SetUpSuite(c *C) {
```

Comment:
why add it?

Reply:
Because to avoid dependency, cannot import `schedulers` any more. So we need to register scheduler names that we'll use in tests.